apps/challenges/views.py

```diff
@@ -106,6 +106,33 @@ def challenge_list(request, challenge_host_team_pk):
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
 
+@throttle_classes([UserRateThrottle])
+@api_view(['GET'])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def challenge_list_participant(request, participant_team_pk):
+    """
+    Returns lists of callenges that the participant team has participated in.
+    """
+    try:
+        participant_team = ParticipantTeam.objects.get(
+            pk=participant_team_pk)
+    except ParticipantTeam.DoesNotExist:
+        response_data = {'error': 'ParticipantTeam does not exist'}
+        return Response(response_data, status=status.HTTP_406_NOT_ACCEPTABLE)
+
+    if request.method == 'GET':
+
+        participant_team = ParticipantTeam.objects.get(
+            pk=participant_team_pk)
```

Comment:
There is no need for querying the database again as you have already received the value in line 118 if it exists. Please remove this line(126-127).